### PR TITLE
[#2532][#2512] fix(IT): fix the web e2e test errors when running on the github actions

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

Fix web e2e test running.

Testing on GitHub actions will result in errors, but it doesn't happen locally. It's probably due to performance issues in the testing environment causing the frontend delay animation to take longer than expected.

### Why are the changes needed?

Fix: #2532
Fix: #2512

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

Github Actions

### CODE REVIEW

**Comment by LauraXia123** [comment]
@ch3yne  If disabled e2e test before fixed, Can you change the title of the PR?

**Comment by jerryshao** [comment]
What is the exact issue here, can we just add more descriptions about the problem and how do we fix it?

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 1]
```diff
@@ -38,6 +40,9 @@ public class MetalakePage extends AbstractWebIT {
   @FindBy(xpath = "//*[@id='submitHandleMetalake']")
   public WebElement submitHandleMetalakeBtn;
 
+  @FindBy(xpath = "//*[@id='cancelHandleMetalake']")
+  public WebElement cancelHandleMetalakeBtn;
+
```
[the remark this answers is not in the document]
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:45]: Done

**Comment by LauraXia123** [comment]
LGTM, Thanks @ch3yne

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java [hunk 2]
```diff
@@ -17,7 +32,61 @@ public class AbstractWebIT extends AbstractIT {
...
+      action.moveToElement(element).click().build().perform();
+      Thread.sleep(ACTION_SLEEP_MILLIS);
+      LOG.error(e.getMessage(), e);
+    }
+  }
+
+  @BeforeEach
+  public void beforeEachTest() {
+    try {
+      Thread.sleep(ACTION_SLEEP_MILLIS);
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java:85]: I think better change variable name `ACTION_SLEEP_MILLIS` to `EACH_TEST_SLEEP_MILLIS`

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 3]
```diff
@@ -89,8 +84,14 @@ public void setMetalakeCommentField(String commentField) {
   }
 
   public void setQueryInput(String queryInput) {
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:86]: I think better change function and variable name `QueryInput` to `QueryParams`.

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 4]
```diff
@@ -263,36 +272,28 @@ public boolean verifyQueryMetalake(String name) {
...
-  //    } finally {
-  //      // Back to homepage
-  //      String xpath = "//*[@data-refer='back-home-btn']";
-  //      WebElement backHomeBtn = driver.findElement(By.xpath(xpath));
-  //      backHomeBtn.click();
-  //    }
-  //  }
+  public boolean verifyLinkToCatalogsPage(String name) {
+    try {
+      String xpath = "//a[@data-refer='metalake-name-link']";
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:282]: Can we define a link element variable in class member?

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 5]
```diff
@@ -211,26 +211,36 @@ public boolean verifyShowMetalakeDetails(String name) {
...
+      clickAndWait(closeDetailsBtn);
     }
   }
 
   public boolean verifyEmptyMetalake() {
-    String xpath = "//div[contains(@class, 'MuiDataGrid-overlay')]";
-    WebElement noMetalakeRows = driver.findElement(By.xpath(xpath));
+    try {
+      String xpath =
+          "//div[@data-refer='metalake-table-grid']//div[contains(@class, 'MuiDataGrid-overlay')]";
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:224]: Can we define a link element variable in class member?

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 6]
```diff
@@ -137,36 +151,22 @@ public void setMetalakeProps(int index, String key, String value) {
...
 
-  public boolean checkIsErrorName() {
-    String xpath = "//div[@data-refer='metalake-name-field']";
-    WebElement nameField = findElementByXPath(xpath);
-    List<WebElement> errorText =
-        nameField.findElements(By.xpath("//div[contains(@class, 'Mui-error')]"));
+  public boolean checkIsErrorName() throws InterruptedException {
+    try {
+      String xpath = "//div[@data-refer='metalake-name-field']";
+      WebElement nameField = driver.findElement(By.xpath(xpath));
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:157]: Can we define a link element variable in class member?

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java [hunk 7]
```diff
@@ -17,7 +32,61 @@ public class AbstractWebIT extends AbstractIT {
...
+      wait.until(ExpectedConditions.visibilityOf(element));
+      wait.until(ExpectedConditions.elementToBeClickable(element));
+
+      element.click();
+      Thread.sleep(ACTION_SLEEP_MILLIS);
+    } catch (ElementClickInterceptedException e) {
+      Actions action = new Actions(driver);
+      action.moveToElement(element).click().build().perform();
+      Thread.sleep(ACTION_SLEEP_MILLIS);
+      LOG.error(e.getMessage(), e);
```
[the remark this answers is not in the document]
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java:78]: There's no need to print error messages here, I removed that line and added a comment.

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java (hunk 2 above)
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/AbstractWebIT.java]: Done

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java (hunk 3 above)
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java]: Done

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java (hunk 4 above)
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java]: Done

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java (hunk 6 above)
Comment (ch3yne) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java]: Done

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request merged**